R/trackhub.R

```diff
@@ -250,7 +250,7 @@ setClass("TrackHubGenome",
 trackhub <- function(x) x@trackhub
 
 getTrackDbContent <- function(x) {
-    TrackContainer(x)
+    # TrackContainer(x)
```

Comment:
Still working on this? It's commented out.

Reply:
Yes, `getTrackDbContent` is a little difficult to implement as R does not support reference or pointer which could be of help in implementing tree.